src/helpers/metrics.ts

```diff
@@ -0,0 +1,110 @@
+import {Dimensions, Platform} from 'react-native'
+
+export type IMetrics = {
+  // Text Size
+  readonly title: number
+  readonly span: number
+
+  // spacing
+  readonly tiny: number
+  readonly xxs: number
+  readonly xs: number
+  readonly small: number
+  readonly sMedium: number
+  readonly medium: number
+  readonly large: number
+  readonly xl: number
+  readonly xxl: number
+  readonly huge: number
+  readonly massive: number
+
+  readonly borderRadius: number
+  readonly borderRadiusLarge: number
+  readonly borderRadiusHuge: number
+
+  // margin
+  readonly marginTop: number
+  readonly marginHorizontal: number
+  readonly marginVertical: number
+  readonly paddingHorizontal: number
+
+  readonly voucherBorderRadius: number
+  readonly logoWidth: number
+  readonly logoHeight: number
+  readonly icon: number
+}
+
+export type IHitSlop = {
+  readonly top: number
+  readonly bottom: number
+  readonly right: number
+  readonly left: number
+}
+
+const hitSlop: IHitSlop = {
+  top: 10,
+  bottom: 10,
+  right: 10,
+  left: 10,
```

Comment:
use `responsiveHeight` for these values

Reply:
The helpers have already been merged in another PR